docs/gettingstarted/structuring.md

```diff
@@ -1,54 +1,79 @@
 ------------------------------------------
 
-Rather than clutter up a single class and package with everything, it is recommended you break your mod into subpackages.
+So, now you have a top level package for your mod and you are ready to get those fingers going. But, before you go and make the Next Best Thing, we should talk about package organization and the use of subpackages. In short, *do it*. Literally nobody like reading a wall of text in a single class trying to figure out how your mod works so they can extend it or fix a bug or whatever.
 
-A common subpackage strategy has packages for `common` and `client` code, which is code that can be run on server/client and client, respectively. Inside the `common` package would go things like Items, Blocks, and Tile Entities (which can each in turn be another subpackage). Things like GUIs and Renderers would go inside the `client` package.
+A common package layout has a `common` subpackage, for client *and* server things, and `client` subpackage, for client-only things. You could also have a subpackage for events and one for items and so forth. An example of this would be:
 
-!!! note
-
-    This package style is only a suggestion, though it is a commonly used style. Feel free to use your own packaging system.
+* `com.examplemod.awesomesauce.api` for your mods API
+* `com.examplemod.awesomesauce.client.gui` for client-side GUI classes
+* `com.examplemod.awesomesauce.events` for your custom events
 
-By keeping your code in clean subpackages, you can grow your mod much more organically.
+Organize it in a way that it makes it easy for you to follow your code, while also allowing your mod to grow and evolve organically as time progresses and new features are added to Minecraft and Forge.
 
 Class Naming Schemes
 --------------------
 
-A common class naming scheme allows easier deciphering of what a class is, and also makes it easier for someone developing with your mod to find things.
-
-For Example:
-
-* An `Item` called `PowerRing` would be in an `item` package, with a class name of `ItemPowerRing`.
-* A `Block` called `NotDirt` would be in a `block` package, with a class name of `BlockNotDirt`.
-* Finally, a `TileEntity` for a block called `SuperChewer` would be a `tile` or `tileentity` package, with a class name of `TileSuperChewer`.
-
-Prepending your class names with what *kind* of object they are makes it easier to figure out what a class is, or guess the class for an object.
+Naming your classes, and being consistant, is important to knowing what those classes do. `BlockSauce` should be a Block, `ItemSpoon` should be an Item, et cetera. However, making sure those classes are in the right package will also help with the ability to find that class if you decide you need to look at it one day down the line. If you have `EntitySauceMonster` in the `com.examplemod.awesomesauce.client.gui` package, you might need to think about refactoring your code to better structure it.
```

Comment:
You need to make note that these are purely suggestions and not real good ones at that. Name things whatever ytou want as long as they start with Upper case and are descriptive enough to tell you what you want to know about the class.